src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin position-arrow {
+  bottom: 50%;
+  left: 50%;
+  border-radius: 30px;
+}
+
+@mixin circle {
+  height: 300px;
+  width: 300px;
+  border-radius: 50%;
+}
```

Comment:
your mixins should be reusable 
add arguments to your mixin, if it circle - call it `size` and pass this size to `width` and `height`